types/react/index.d.ts

```diff
@@ -3326,7 +3349,7 @@ declare global {
             foreignObject: React.SVGProps<SVGForeignObjectElement>;
             g: React.SVGProps<SVGGElement>;
             image: React.SVGProps<SVGImageElement>;
-            line: React.SVGProps<SVGLineElement>;
+            line: React.SVGLineElementAttributes<SVGLineElement>;
```

Comment:
q: why this new interface was added? I certainly don't mind it - I'm just curious

Reply:
So that `react-blessed` can add attributes. Ultimately, we should just add separate prop interfaces for all elements. I'm just a bit hesitant though because people may not understand what goes in there and when. There's a common confusion around HTML attributes vs DOM properties vs React props.